libkbfs/file_data.go

```diff
@@ -38,9 +38,14 @@ func newFileData(file path, uid keybase1.UID, crypto cryptoPure,
 	}
 }
 
+type parentBlockAndChildIndex struct {
```

Comment:
This type could use a comment, mentioning that pblock is an indirect block, and `childIndex` is an index into `pblock.IPtrs`. 

I also tried to think of a better name, but didn't really come up with anything. Oh, well.